libraries/ESP8266WiFiMesh/src/ESP8266WiFiMesh.h

```diff
@@ -24,52 +24,193 @@
 #include <WiFiClient.h> 
 #include <WiFiServer.h>
 #include <functional>
+#include <vector>
+#include "NetworkInfo.h"
+#include "TransmissionResult.h"
+
+#define ENABLE_STATIC_IP_OPTIMIZATION // Requires Arduino core for ESP8266 version 2.4.2 or higher and lwIP2 (lwIP can be changed in "Tools" menu of Arduino IDE).
+#define ENABLE_WIFI_SCAN_OPTIMIZATION // Requires Arduino core for ESP8266 version 2.4.2 or higher. Scan time should go from about 2100 ms to around 60 ms if channel 1 (standard) is used.
 
 class ESP8266WiFiMesh {
 
 private:
-	String _ssid;
-	String _ssid_prefix;
-	uint32_t _chip_id;
+  String _ssid;
+  String _mesh_name;
+  String _node_id;
+  int _server_port;
+  String _mesh_password;
+  uint8 _mesh_wifi_channel;
+  bool _verbose_mode;
+  WiFiServer _server;
+  uint32_t _lwip_version[3];
+  static const uint32_t lwip_version_203_signature[3];
+  String _message = "";
 
-	std::function<String(String)> _handler;
-	
-	WiFiServer  _server;
-	WiFiClient  _client;
+  static String last_ssid;
+  static bool static_IP_activated;
+  static IPAddress static_IP;
+  static IPAddress gateway;
+  static IPAddress subnet_mask;
 
-	void connectToNode(String target_ssid, String message);
```

Comment:
I suggest typedefing (or using = ) these definitions, and doing so within the class:
...
using requestHandlerType = std::function<String(String, ESP8266WiFiMesh *)>;
...

The type can then be used and reused without duplication as:
ESP8266WiFiMesh::requestHandlerType someRequestHandler...